Cargo.toml

```diff
@@ -25,20 +25,21 @@ description = "A pure-Rust implementation of Bulletproofs using Ristretto"
 edition = "2018"
 
 [dependencies]
+byteorder = { version = "1", default-features = false }
+clear_on_drop = { version = "0.2", default-features = false }
```

Comment:
Don't order these, as it will make taking upstream changes more annoying.

Reply:
Done